Dalamud/Interface/Internal/TextureManager.cs

```diff
@@ -480,6 +499,10 @@ public TextureManagerPluginScoped(TextureManager textureManager)
         return wrap;
     }
 
+    /// <inheritdoc/>
+    public string? GetIconPath(uint iconId, ITextureProvider.IconFlags flags = ITextureProvider.IconFlags.HiRes, ClientLanguage? language = null)
```

Comment:
This function does the exact same as the one above no?

Reply:
No. This one just exists as a convenience function, if you need to do something else with an icon(e.g. load a tex file to send somewhere).